faith_presentation/src/commonMain/kotlin/net/thechance/mena/faith/presentation/base/BasePagingSource.kt

```diff
@@ -0,0 +1,54 @@
+package net.thechance.mena.faith.presentation.base
+
+import androidx.paging.Pager
+import androidx.paging.PagingConfig
+import androidx.paging.PagingData
+import androidx.paging.PagingSource
+import androidx.paging.PagingState
+import kotlinx.coroutines.flow.Flow
+
+internal class BasePagingSource<T : Any>(
+    private val onError: (Throwable) -> Unit = {},
+    private val onFetchPage: suspend (Int, Int) -> List<T>
```

Comment:
It is clearer to give meaningful names to lambda parameters.

```suggestion
    private val onFetchPage: suspend (pageNumber: Int, pageSize: Int) -> List<T>
```